ci/download_xs.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ ! -d $HOME/endf71_multitemp/ ]; then
```

Comment:
I've discovered the hard way that this doesn't work because even if the cached data wasn't there, Travis still creates this directory. To get around this, you can either check `-e` on a file within the directory or `-d` on a subdirectory, e.g.
```bash
if [ ! -d $HOME/endf71_multitemp/photon/ ]; then
```